src/pages/_app.tsx

```diff
@@ -52,6 +52,9 @@ export default function App({ Component, pageProps }: AppProps) {
 
   // Track page view events to Adobe Analytics
   useEffect(() => {
+    console.log("ADOBE_EMBED_URL: ", process.env.ADOBE_EMBED_URL);
```

Comment:
Ah, so you can't access `process.env` on the client side (useEffect runs on the client) because that's a server side only global variable (in node). So this _should_ be accessed through appConfig instead. If it's not working then update `ADOBE_EMBED_URL` to `NEXT_PUBLIC_ADOBE_EMBED_URL` everywhere. It's considered "unsafe" but in this case, we do want this to be sent to the client so it's okay to do this.

Reply:
can I add this line to the utils file, since that runs on the server?

Reply:
Yes, exactly, that's the first place where the env var will be read.

Reply:
cool, donezo

Reply:
Did you push anything new?

Reply:
i forgot to push it up oops

Reply:
it's there now